### PR TITLE
fix: fix gradio web server

### PR DESCRIPTION
**Pull request by learnerljh**
# Description

Describe your changes in detail.

## Motivation and Context

Why is this change required? What problem does it solve?
If it fixes an open issue, please link to the issue here.
You can use the syntax `close #1314520` if this solves the issue #15213

- [ ] I have raised an issue to propose this change ([required](https://github.com/PKU-Alignment/align-anything/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://github.com/PKU-Alignment/align-anything/blob/HEAD/.github/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [x] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -288,6 +288,37 @@ class PKUSafeRLHF(Template):
...
+
+### 启动 Gradio Web 服务器
+```Shell
+python -m align_anything.serve.gradio_web_server --controller http://localhost:10000 --model-list-mode reload
+```
+你现在已经启动了 Gradio Web 界面。接下来，你可以使用屏幕上打印出的 URL 打开 Web 界面。你可能会注意到目前还没有列出任何模型，不用担心，因为我们还没有启动任何模型工作者。一旦启动了模型工作者，模型列表将会自动更新。
+
+### 启动模型工作者
+
+这是实际执行 GPU 推理的 *工作者*。每个工作者负责一个指定在 `--model-path` 中的单一模型，并且请参考 `align_anything/configs` 中的 `template.py` 文件找到相应的模板名称。
```
Comment (Gaiejj) [inline README_zh-CN.md:307]: 工作者 -> worker

**Review by Gaiejj — APPROVED** [review]
LGTM.

**Review by htlou — APPROVED** [review]
LGTM

**Review by XuyaoWang — APPROVED** [review]
good code

**Comment by learnerljh** [comment]
Thank you!

**Pull request merged**